modules/mobile/docs/schemas/SecureMessageReplyRequest.yml

```diff
@@ -1,6 +1,22 @@
 type: object
 required:
+  - recipient_id 
+  - category
+  - subject
   - body
 properties:
+  recipient_id:
```

Comment:
Actually I was wondering if recipient_id is required for replies, or if there's validation that the recipient matches the sender of the replied-to message. That would be good to clarify.

Reply:
In my testing, I've found that aside from file uploads, only `body` is required/allowed for replies.  `recipient_id` is ignored on the MHV side.  I'll make an update to our API's validation.